docs/configuration.rst

```diff
@@ -0,0 +1,53 @@
+*************
+Configuration
+*************
+
+Config system parameters
+========================
+
+Some Loihi-specific configuration options
+are exposed through Nengo's config system.
+
+To configure these parameters,
+first call `.add_params` on your network.
+Then, you can use the added configuration options in your network::
+
+  with nengo.Network() as net:
+      nengo_loihi.add_params(net)
+
+      a = nengo.Ensemble(10, 1)
+      net.config[a].on_chip = False  # run the ensemble off-chip
+
+For a full list of available config options, see `.add_params`.
+
+Build parameters
+================
```

Comment:
That's not a bad idea. I was also wondering whether there's a way to do the `Model` docstring so that configurable parameters are in a different section or something, so that it's clear which ones you might want to change. Either way, I agree that they need to be enumerated somewhere.

Reply:
Done in aaeebaa; it looks like this:

![build-params](https://user-images.githubusercontent.com/203709/55347262-60525180-5482-11e9-99f1-2529ed9222ee.png)

Not super great (it's clearly interpreting the "Build parameters" thing as an attribute with no documenation) but maybe better than having a separate list of build parameters in the main part of the docstring?

Reply:
Yeah, it looks all right. Good enough for now, IMO.

Reply:
Yeah I feel like numpydoc is kind of constraining us here and doing it better might require a switch to Napoleon or something... which is a bigger change so later.